ts/features/it-wallet/saga/ItwSagaUtils.ts

```diff
@@ -0,0 +1,29 @@
+import { CommonActions } from "@react-navigation/native";
+import { isActionOf } from "typesafe-actions";
+import { put, take } from "typed-redux-saga/macro";
+import NavigationService from "../../../navigation/NavigationService";
+import {
+  identificationRequest,
+  identificationSuccess
+} from "../../../store/actions/identification";
+import I18n from "../../../i18n";
+
+/**
+ * Generator function that handles the PIN verification.
+ * Throws an error if the identification fails.
+ */
+export function* verifyPin() {
```

Comment:
Is this an util function to add identification modal screen during a flow?

Reply:
Yes it is but in a more generic way it just verifies the user PIN in any saga.